tools/clang/test/HLSLFileCheck/hlsl/workgraph/wavesize_range.hlsl

```diff
@@ -0,0 +1,120 @@
+// RUN: %dxc -T lib_6_8 %s | FileCheck %s
+
+// Check the WaveSize attribute is accepted by work graph nodes
+// and appears in the metadata
```

Comment:
This looks like diagnostics verification, belonging under a different path using -verify, no?